jenkins/check_copyright.sh

```diff
@@ -59,7 +61,7 @@ if [ -e $ERRFILE ]; then # if error file exists then there were errors
     >&2 echo "--- ERRORS ---"
     cat $ERRFILE 1>&2 # send output to stderr so it can be redirected as error if desired
     >&2 echo "--------------"
-    exit 1 # tell the caller there was an error (so Jenkins will fail the CI task)
+    exit 1 # tell the caller there was an error (so the CI task will fail)
```

Comment:
The `exit 1` is still here

Reply:
Weird, I guess the previous view pushed the new line low and I did not see it. Sorry about that.